thrift-interface-descriptions/component-cpis/registry-api.thrift

```diff
@@ -869,11 +870,15 @@ service RegistryService {
              * @param gatewayId
              *    ID of the gateway which need to list all available application deployment documentation.
              *
+             * @param userName
```

Comment:
Likewise please update the method documentation.

Reply:
Done.